packages/wdio-selenium-standalone-service/src/launcher.ts

```diff
@@ -82,10 +82,35 @@ export default class SeleniumStandaloneLauncher {
         const capabilities = (Array.isArray(this._capabilities) ? this._capabilities : Object.values(this._capabilities))
         for (const capability of capabilities) {
             const cap = (capability as Options.WebDriver).capabilities || capability
-            const c = (cap as Capabilities.W3CCapabilities).alwaysMatch || cap
 
-            if (!isCloudCapability(c) && hasCapsWithSupportedBrowser(c)) {
-                Object.assign(c, DEFAULT_CONNECTION, { ...c })
+            /**
+             * handle standard mode vs multiremote mode, e.g.
+             * ```js
+             * capabilities: [{
+             *   browserName: 'chrome',
+             *   hostname: 'localhost'
+             * }]
+             * ```
+             * vs.
+             * ```js
+             * capabilities: {
+             *   myBrowser: {
+             *     hostname: 'localhost',
+             *     capabilities: { browserName: 'chrome' }
+             *   }
+             * }
+             */
+            const remoteCapabilities = (cap as Capabilities.W3CCapabilities).alwaysMatch || cap
+            const objectToApplyConnectionDetails = Array.isArray(this._capabilities)
```

Comment:
what bothers me a bit is that we already did a check for the type of capabilities at line 82, what do you think about something like this?
```
Array.isArray(this._capabilities) ? this._capabilities : Object.values(this._capabilities))

if (Array.isArray(this._capabilities)) {
  // logic for standard mode
} else {
  // logic for multiremote mode
}
```
It would prevent us from having multiple checks and is easy to follow when reading